### ISSUE TITLE
[Question]: Multiple VPN sessions in one configuration file

### ISSUE THREAD

**Issue by dclong** [sshuttle/sshuttle#539]
Is it possible to configuration multiple VPN sessions in one configuration file? I couldn't find any doc on this part.

**Comment by skuhl** [comment]
No, but I think you can run sshuttle multiple times with different config files.

**Comment by rayjanoka** [comment]
I'm not sure how well this works but, https://github.com/albertodonato/sshoot
